org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/BackendUtils.java

```diff
@@ -1054,6 +1054,25 @@ public static void copyIdToFields(HasIdInIdOrNameField element, DefinitionsChild
         }
     }
 
+    public static void fixReferences(Definitions element, DefinitionsChildId id) {
```

Comment:
What are you doing here? There is already a method which does something similar: `BackendUtils::synchronizeReferences`

Reply:
Please check this here again

Reply:
I'll check this in the weekend